internal/controller/datadogagent/feature/orchestratorexplorer/feature_test.go

```diff
@@ -142,7 +142,17 @@ func orchestratorExplorerClusterAgentWantFuncV2() *test.ComponentTest {
 			customConfig := v2alpha1.CustomConfig{
 				ConfigData: apiutils.NewStringPointer(customConfDataV2),
 			}
-			hash, err := comparison.GenerateMD5ForSpec(&customConfig)
+			truePointer := true
```

Comment:
```suggestion
			trueValue := true
```

we do it like this elsewhere

Reply:
https://github.com/DataDog/datadog-operator/pull/1340/commits/c3fcffee51686d555c957949fda69fa35dfa48c2